### ISSUE TITLE
[Feature] Enchantable shulker compat

### ISSUE THREAD

**Issue by Austin001316** [Aton-Kish/reinforced-shulker-boxes#43]
This mod's shulker boxes currently arent enchantable when enchanted shulkers installed, I would like to request compatibility

Here is a link to the mod https://github.com/RubixDev/EnchantedShulkers#configuration

**Comment by RubixDev** [comment]
@Aton-Kish In case you need help with that, feel free to contact me here or on Discord (RubixDev#7875). I have a very basic overview of what to do [in my README](https://github.com/RubixDev/EnchantedShulkers#for-mod-developers).

**Comment by Aton-Kish** [comment]
Hi,

I added portable container tags and confirmed that Siphon and Refill were available in v2.2.0.
But I haven't been able to support glint yet because I want to keep loose coupling between both mods.

**Comment by RubixDev** [comment]
@Aton-Kish I finally got around to have a look on how to support the glint for reinforced shulkers, and I was able to get it all working by mixing into two of your classes when both mods are loaded: https://github.com/RubixDev/EnchantedShulkers/commit/8ca24cebac56dd5b064b279e548c9c95f429d686 (so no further action on your side will be required).

For achieving non-overlapping glint on placed closed shulkers, I also had to copy all your shulker textures and rearrange them (just like I had to with the vanilla textures). This means your textures will [from now on also be in EnchantedShulkers](https://github.com/RubixDev/EnchantedShulkers/tree/8ca24cebac56dd5b064b279e548c9c95f429d686/src/main/resources/assets/enchantedshulkers/textures/entity/reinforced_shulker). I hope this is okay with you.
